clang/lib/Driver/Driver.cpp

```diff
@@ -5606,7 +5612,7 @@ class OffloadingActionBuilder final {
 
             ActionList TformInputs{PostLinkAction, ForEachWrapping};
             auto *ReplaceFilesAction = C.MakeAction<FileTableTformJobAction>(
-                TformInputs, types::TY_Tempfiletable, types::TY_Tempfiletable);
+                TformInputs, types::TY_Filetable, types::TY_Filetable);
```

Comment:
This doesn't look to be involved with any 'spv' files for cleanup.  Is this needed?